src/app/lib/utilities/getToggles/index.js

```diff
@@ -32,7 +34,14 @@ const getToggles = async (service, cache) => {
 
   try {
     logger.info(CONFIG_REQUEST_RECEIVED, { url, service });
+    const startHrTime = process.hrtime();
```

Comment:
Are we sure this never runs in browser?  Will skip toggles fetch if so as `process` will be undefined

Reply:
Great observation @ryanmccombe! We can confirm that the `getToggles` utility only runs server-side. 

Also, there are 2 other places within `getToggles` where we have been using `process`.